### PR TITLE
Add sorting feature to system

### LINKED ISSUE #5: Add sorting feature into system
Edit code such that with each addition of a person into the list, it would sort them according to their names.

### PR DESCRIPTION
**Pull request by TiankaiChen**
Resolves #5 . Please review.

### CODE REVIEW

File: src/seedu/addressbook/data/person/NameComparator.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+ //as;
+import java.util.Comparator;
+
+public class NameComparator implements Comparator<Person>{
```
Comment (rickyyx) [inline src/seedu/addressbook/data/person/NameComparator.java:5]: Can do an inline one as well if it is only used once. This is fine of course.

**Review by rickyyx — COMMENTED** [review]
Did you add the command word? Or there is missing commits?

File: src/seedu/addressbook/data/person/NameComparator.java (hunk 1 above)
Comment (daredevil999) [reply src/seedu/addressbook/data/person/NameComparator.java:5]: Looks good

**Review by daredevil999 — APPROVED** [review]
Looks good. The sorting is automatically done after a new person is added. This makes sense, don't think anyone will want to see a list of people that is unsorted. So having the default list be a sorted one is good

**Pull request merged**